spec/order_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../lib/order'
+
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:basket) { { pasta: 6,  gelato: 4 } }
+  
+  # it { is_expected.to respond_to(:add_to_order).with(1).argument }
+  
+  describe "#add_to_order" do
+    it 'adds a dish to the order' do
+      order.add_to_order(:pasta, 6)
```

Comment:
Suggestion: If the sample code for these examples do not need to be differnet, you could considered using a before block to minimise duplication of test set up code.